doc/engineering-reference/src/advanced-surface-concepts/green-roof-model-ecoroof.tex

```diff
@@ -323,7 +323,7 @@ \subsection{Final Equations}\label{final-equations}
 
 \subsection{Green Roof Nomenclature}\label{green-roof-nomenclature}
 
-C\(_{1}\), C\(_{2}\), C\(_{3}\)~~~ = ~~~~~~~~~ coefficients in linearized temperature equations
+C\(_{1}\), C\(_{2}\), C\(_{3}\)~~~~~~ = ~~~~~~~~~ coefficients in linearized temperature equations
```

Comment:
@RKStrand as you work through these, feel free to remove all these goofy forced spacing characters.  We have the `align` environment to do that if we want.  I know that adds to the amount you do, so I'm really just saying if you do have to touch a group of equations to fix alignment, make the change to the align environment.  You don't have to fix every single one though.

Reply:
@Myoldmopar I have not used align really.  In many of these cases, I am just getting rid of all of the forced spaces.  There were exceptions when I thought the spacing looked like it should.  However, in situations where there was a list of terms from equations after a "where:" I started eliminating all of the forced spacing because it never lined up properly/was a total pain to even try it once.  It may be that not every section is exactly identical, but what we end up with should be much more standardized and fairly consistent (I hope).

Reply:
Yeah, I hesitated before commenting but went ahead and wrote it.  Let me try to clarify a couple things:
 - The `align` environment is just another way to align equations up.  Using an `equation` environment, with a nested `array` environment is totally fine also.  Keep on using that.
 - As for the spacing itself, I just noticed that in this section, you added some extra tildes to try to line things back up.  In general, if you are already modifying a chunk of code that is aligned using these tildes, please go ahead and make them properly aligned environments as you have elsewhere.  These tildes just came from the original Microsoft Word to Markdown conversion, and should eventually be cleaned out.

Nothing to do here for now though, there's a ton of good fixes here.